src/SshNetTests/SftpTests.cs

```diff
@@ -4426,6 +4447,400 @@ public void Sftp_SftpFileStream_Seek_BeyondEndOfFile()
             }
         }
 
+        [TestMethod]
+        public void Sftp_SftpFileStream_Seek_BeyondEndOfFile_SeekOriginEnd()
+        {
+            const string remoteFile = "/home/sshnet/test";
+
+            using (var client = new SftpClient(_connectionInfoFactory.Create()))
+            {
+                client.BufferSize = 500;
+                client.Connect();
+
+                if (client.Exists(remoteFile))
+                {
+                    client.DeleteFile(remoteFile);
+                }
+
+                try
+                {
+                    // create single-byte file
+                    using (var fs = client.OpenWrite(remoteFile))
```

Comment:
Nit: in my own code, I try not to use uncommon acronyms as they can lead to confusion, while almost all modern IDEs provide easy auto-completion for coding velocity.
```suggestion
                    using (var fileStream = client.OpenWrite(remoteFile))
```

Reply:
I prefer to keep this as I think it's fairly common.
OK?

Reply:
As a 'nit' it is always OK not to change.

Reply:
If I weren't interested in your remarks, I wouldn't be asking for a review.
So no, I don't think it's OK to plainly ignore remarks.
I strongly prefer there's a concensus, but I'm not sure that not always possible.
When not, there should at least be some kind of motivation for the decision that was made.